Containers/mastercontainer/Dockerfile

```diff
@@ -19,31 +27,46 @@ RUN mkdir -p /var/www/docker-aio;
 
 WORKDIR /var/www/docker-aio
 
-RUN apt-get update; \
-    apt-get install -y --no-install-recommends \
-        git \
+RUN set -ex; \
+    apk add --no-cache 
+        ca-certificates \
+        wget \
+        tzdata \
+        bash \
+        apache2 \
```

Comment:
is it really called apache and not http?

Reply:
yes: https://pkgs.alpinelinux.org/package/v3.17/main/x86_64/apache2